notes/2023/2023-11-30.md

```diff
@@ -0,0 +1,42 @@
+# 2023-11-30 ESLint TSC Meeting Notes
+
+## Transcript
+
+[`2023-11-30-transcript.md`](2023-11-30-transcript.md)
+
+## Attending
+
+* Nicholas C. Zakas (@nzakas) - TSC
+* Milos Djermanovic (@mdjermanovic) - TSC
+
+@nzakas moderated, and @sam3k took notes.
+
+## Topics
+
+### [v9.0.0 Status](https://github.com/orgs/eslint/projects/4/views/2) 
+
+| Feature | Status | Notes |
+| ------- | -- | -- |
+| [Change Request: Drop Node v12/v14/v16/v17/v19 support](https://github.com/eslint/eslint/issues/17595) | PR is ready for merge | @nzakas and @mdjermanovic will set status to ready for merge |
+| [Change Request: Remove all formatters except stylish, html, json, and json-with-meta](https://github.com/eslint/eslint/issues/17524) | Waiting on the last remaining changes from @mdjermanovic | |
+| [Change default rule schema to `schema: []` and drop support for function-style rules](https://github.com/eslint/eslint/issues/14709) | @mdjermanovic opened [draft PR](https://github.com/eslint/eslint/pull/17792) | |
+| [Change Request: stop running rules set to "warn" when `--quiet` is used](https://github.com/eslint/eslint/issues/16450) | Close to done; just making final changes. | |
+| [Omiting any file/dir positional parameter to CLI does not give an error](https://github.com/eslint/eslint/issues/14308) | Ready for review | @mdjermanovic will review soon |
```

Comment:
Is this ready for review or ready to implement? Issue status says "ready to implement" and the PR linked to this issue is in [draft](https://github.com/eslint/eslint/pull/17644). 

Let me know and I'll update the status of the issue if needed.

Reply:
The status should be "Implementing" :+1: Thanks for catching that.